### ISSUE TITLE
NTR: Lysophosphatidylcholine (LPC) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8388]
New term request Lysophosphatidylcholine (LPC) high in blood (elthat_150829142959):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000403
- label: Elevated circulating lysophosphatidylcholine concentration
- parents: [Abnormal circulating free fatty acid concentration (HP:0040300)](https://hpo.jax.org/app/browse/term/HP:0040300)
- pmid: PMID:32717772
- def: The concentration of lysophosphatidylcholines in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8388
- synonyms:
